### DISCUSSION TITLE
System & network requirements for running

### DISCUSSION THREAD

**Discussion by jimmymcheung** [docker-mailserver/docker-mailserver#2452] [Q&A] [answered]
Will this docker image be run on a 32bit cpu (arm32v7, 4 core)? 
and 
Does the server use the 80 or 443 port, since they are occupied by web server.

**Comment by georglauterbach** [comment, marked as answer]
> Will this docker image be run on a 32bit cpu (arm32v7, 4 core)?

We're building for `linux/amd64`, `linux/arm/v7` and `linux/arm64`. So yes, `linux/arm/v7` should be the 32bit ARM architecture.

> Does the server use the 80 or 443 port, since they are occupied by web server.

No, these ports are (as you already noted) for web servers, not for mail servers. DMS mainly uses the ports described in `docker-compose.yml` (i.e. 143, 465, 587, 994).

**Reply by jimmymcheung** [reply]
Thanks
